resources/wikia/modules/geo.js

```diff
@@ -85,7 +85,12 @@
 		 * @returns {boolean}
 		 */
 		function isProperCountry(countryList) {
-			return !!(countryList && countryList.indexOf && countryList.indexOf(getCountryCode()) > -1);
+			return !!(
+				countryList &&
+				countryList.indexOf &&
+				countryList.indexOf(getCountryCode()) > -1 &&
+				countryList.indexOf('non-' + getCountryCode()) === -1
```

Comment:
What do you think about keeping `'non-'` in a `negativePrefix` variable defined at the top together with `cookieName` and `geoData`? Will it make code less readable?